Tracking/DisasterTrackerApp.Models/Disaster/EventDto.cs

```diff
@@ -1,13 +1,30 @@
+using System.Text.Json.Serialization;
+
 namespace DisasterTrackerApp.Models.Disaster;
 
 public class EventDto
```

Comment:
```suggestion
public class DisasterEventDto
```
to avoid confusion between calendar events and disaster events